roles/ssh_host_signer/tasks/main.yml

```diff
@@ -1,7 +1,9 @@
 ---
 - name: 'Set hostname to inventory_hostname.'
   hostname:
-    name: "{{ inventory_hostname | regex_replace('^' + ai_jumphost + '\\+','') }}"
+    name: "{%- if groups['irods'] is defined and inventory_hostname in groups['irods'] and
```

Comment:
This will work, but is very specific for one type of machine from one type of inventory group. Would it be possible to make this more generic? Putting fqdn values in ```group_vars/nibbler_cluster/ip_addresses.yml``` was not the issue, the issue was listing the wrong fqdn for a certain IP address. The other issue is that these machines can have different interfaces with different names, but there van be only one hostname. Hence I think we should look for a more generic way to specify hostnames in fqdn syntax for individual machines.
In theory Ansible can also use a ```host_vars``` subdir in addition to ```group_vars```, but the problem is that the hostnames can change dynamically when using a dynamic inventory to handle different jumphosts. Maybe we can specify the fqdn hostname in the static inventory, but will probably require updates to the script that parses the static inventory to create a dynamic one...